web_client/views/popover/AnnotationContextMenu.js

```diff
@@ -39,6 +53,53 @@ const AnnotationContextMenu = View.extend({
         annotation.elements().remove(element);
         this.reset();
         this.trigger('h:close');
+    },
+    _editElements(evt) {
+        evt.preventDefault();
+        evt.stopPropagation();
+
+        const { annotation, element } = this._hovered;
+        this.trigger('h:edit', annotation.elements().get(element));
+    },
+    _setStyleDefinition(group) {
+        const styles = new StyleCollection();
+        return styles.fetch().done(() => {
+            const style = styles.get({ id: group || 'default' });
```

Comment:
If I have an annotation with a style group that no longer exists, the subsequent line with `style.toJSON()` with fail.  Perhaps this should be changed to `const style = styles.get({id: group}) || styles({id: default})`